internal/util/util.go

```diff
@@ -353,3 +354,15 @@ func CallStack() string {
 
 	return string(stack)
 }
+
+// IsVersionSatisfied check if available version satisfies required version
+func IsVersionSatisfied(required, available string) bool {
```

Comment:
Please add a unit test for this function in `internal/util/util_test.go`

Reply:
Also, the commit message mentions "package versions"... what packages? Are these RPM, Golang or other versions?

Reply:
https://github.com/hashicorp/go-version#version-parsing-and-comparison

Should I rather stick to the binary version rather than the package version?

Reply:
Done now, changed it to "util: provide utility to parse and compare versions"